prac_07/myguitars.py

```diff
@@ -6,20 +6,68 @@
 """
 import csv
 from guitar import Guitar
-from operator import itemgetter
 
 
 def main():
-    guitars = []
+    guitars = []  # set guitars to the empty list
+    open_file(guitars)
+    name = input("New guitar name: ")
+    get_new_guitar(guitars, name)  # create Guitar object
+    guitars.sort()
+    print_guitars(guitars)
+    write_file(guitars)
+
+
+def write_file(guitars):
```

Comment:
This function seems pretty good :)
Maybe consider naming it something like save_file?
Both are good names I just think save_file makes more logical sense then write_file